README.md

```diff
@@ -2,11 +2,11 @@
 
 Experimental package with helpful functions for working with Cmd.
 
-This package adds some functions that helps you with constructing of `Cmd msg` from given `msg` and theirs manipulation.
+This package adds some functions that helps you with constructing of `(model, Cmd msg)` pair from given `model` and `msg` and theirs manipulation.
```

Comment:
this is also not acurate. perform attempt and maybe are not working with pair.

there are 2 things that this package provides:

1. constructors for Cmds
2. way to compose (model, Cmd msg) pairs in pipelines.